### PR TITLE
fix kbfs path regex in client

### PR DESCRIPTION
**Pull request by jakob223**
cc @keybase/y2ksquad 
This is the client fix corresponding to https://github.com/keybase/keybase/pull/4571

Issue: Y2K-868

### CODE REVIEW

**Comment by zapu** [comment]
https://github.com/keybase/client/blob/master/go/libkb/assertion_parser.go#L78

something to look at

i don't know how to use assertion_parser code in there though, I'm not familiar with kbfsPathInnerRegExp

**Comment by jakob223** [comment]
CI fail #3 is unrelated chat test. Rebuilding

File: go/libkb/util.go [hunk 1]
```diff
@@ -1065,7 +1065,9 @@ func FindPreferredKBFSMountDirs() (mountDirs []string) {
 }
 
 var kbfsPathInnerRegExp = func() *regexp.Regexp {
-	const socialAssertion = `[-_a-zA-Z0-9.]+@[a-zA-Z.]+`
+	const regularAssertion = `[-_a-zA-Z0-9.+]+@[a-zA-Z.]+`
+	const atContainingAssertion = `\[[-_a-zA-Z0-9.]+@[-_a-zA-Z0-9.]+\]@[a-zA-Z.]+`
```
Comment (songgao) [inline go/libkb/util.go]: Does this only cover the email case? If so would it be better to just put that in the regex?

Also I know I didn’t put any comment before, but would you mind adding some examples in the comment for the new case?

Comment (jakob223) [reply go/libkb/util.go]: I was trying to be mildly future proof / the server accepts assertions that match this regex so I figured we should here too. Will add comments.

**Comment by jakob223** [comment]
CI failed because yarn segfaulted.. previous run where only comments were different passed, ignoring.

**Pull request merged**